### PR TITLE
translate(german): translate blogpost about kepler

### PR DESCRIPTION
**Pull request by thelooter**
Translates the 2023 Blogpost about embracing Kepler "2023-embracing-edge-computing-with-kepler" into German

### CODE REVIEW

**Comment by thelooter** [comment]
The link the Quality Checks are complaining about is the same one as in the original blogpost. It just isn't reachable anymore

File: website/content/de/blog/2023-10-embracing-edge-computing-with-kepler.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Die architektonische Brillanz von Kepler
+
+Die Architektur von Kepler erfasst effizient Metriken auf Pod-/Container-/Prozessebene. Der OTEL-Collector fungiert als Vermittler, verarbeitet und leitet Daten an Ziele weiter. Durch die Nutzung von Prometheus als Backend, kombiniert mit der Leistungsfähigkeit von Grafana, wird Rohdaten in aussagekräftige Visualisierungen umgewandelt. Diese Synergie bietet ein nahtloses Erlebnis und ermöglicht es den Nutzern, komplexe Daten mühelos zu verstehen.
+
+Abschließend lässt sich sagen, dass Kepler zusammen mit OpenTelemetry und seinem Fokus auf Edge Computing einen bedeutenden Sprung im Bereich der Leistungsüberwachung darstellt. Folgen Sie [diesem Link](https://github.com/husky-parul/otel-observability) für die Bereitstellungsschritte. Bleiben Sie dran für weitere Updates zur [Instrumentation des Kepler OTEL SDK](https://github.com/sustainable-computing-io/kepler/issues/659).
+
+Referenz: [Hybrid Cloud Patterns](https://hybrid-cloud-patterns.io/learn/about/)
```
Comment (guidemetothemoon) [inline website/content/de/blog/2023-10-embracing-edge-computing-with-kepler.md:34]: @thelooter thank you for the effort on this! I think the outdated link has been replaced with following one: https://validatedpatterns.io/learn/about-validated-patterns/

Would it be possible to update it here, at least for this version? I can update the other translations afterwards if needed.

Thanks! 🙌

**Review by guidemetothemoon — CHANGES_REQUESTED** [review]

File: website/content/de/blog/2023-10-embracing-edge-computing-with-kepler.md (hunk 1 above)
Comment (thelooter) [reply website/content/de/blog/2023-10-embracing-edge-computing-with-kepler.md:34]: I can update both

Comment (guidemetothemoon) [reply website/content/de/blog/2023-10-embracing-edge-computing-with-kepler.md]: LGTM, thank you!💚

**Pull request merged**